slick/src/main/scala/slick/jdbc/DerbyProfile.scala

```diff
@@ -240,6 +243,27 @@ trait DerbyProfile extends JdbcProfile {
       override def sqlType = java.sql.Types.BINARY
       override def sqlTypeName(sym: Option[FieldSymbol]) = "CHAR(16) FOR BIT DATA"
     }
+
+    class InstantJdbcType extends super.InstantJdbcType {
+      // Derby has no timestamp with timezone type and so using strings as timestamps are
+      // susceptible to DST gaps twice a year
+      override def sqlType: Int = java.sql.Types.VARCHAR
+      override def setValue(v: Instant, p: PreparedStatement, idx: Int): Unit = {
+        p.setString(idx, v.toString)
+      }
+      override def getValue(r: ResultSet, idx: Int): Instant = {
+        r.getString(idx) match {
+          case null => null
+          case instantString => Instant.parse(instantString)
+        }
+      }
+      override def updateValue(v: Instant, r: ResultSet, idx: Int): Unit = {
+        r.updateString(idx, v.toString)
+      }
+      override def valueToSQLLiteral(value: Instant) = {
+        s"'${value.toString}'"
```

Comment:
Here the `toString` of Instant is defined as returning a string in the UTC-timezone.